dev/ts_image.py

```diff
@@ -13,22 +12,32 @@
 """
 
 # Parse command line arguments
+import warnings
+import os
+
+from astropy.utils.exceptions import AstropyDeprecationWarning
+warnings.filterwarnings('ignore', category=AstropyDeprecationWarning)
```

Comment:
If this is brand-new code, is it not dangerous to ignore astropy depreciation warnings? If you use a feature that's deprecated already, there is a good chance it will be removed in one of the next astropy versions.